Formula/scs.rb

```diff
@@ -0,0 +1,39 @@
+class Scs < Formula
+  desc "Conic optimization via operator splitting"
+  homepage "https://web.stanford.edu/~boyd/papers/scs.html"
+  url "https://github.com/cvxgrp/scs/archive/2.1.1.tar.gz"
+  sha256 "0e20b91e8caf744b84aa985ba4e98cc7235ee33612b2bad2bf31ea5ad4e07d93"
+
+  resource "scs-test-data-small-random-socp" do
```

Comment:
Is this part of the release tarball? If so, we could install the file as part of the `install`:

```
pkgshare.install "test/data/small_random_socp"
```

so we don't have to download it again for tests

Reply:
Done.